omc3/model/model_creators/lhc_model_creator.py

```diff
@@ -399,16 +400,32 @@ class LhcBestKnowledgeCreator(LhcModelCreator):  # -----------------------------
     jobfile: str = JOB_MODEL_MADX_BEST_KNOWLEDGE
     files_to_check: tuple[str] = (TWISS_BEST_KNOWLEDGE_DAT, TWISS_ELEMENTS_BEST_KNOWLEDGE_DAT)
 
-    def check_options(self, opt) -> bool:
+    def prepare_options(self, opt) -> bool:
         accel: Lhc = self.accel
-        if accel.list_b2_errors:
-            errors_dir = AFS_B2_ERRORS_ROOT / f"Beam{accel.beam}"
-            for d in errors_dir.iterdir():
-                if d.suffix==".errors" and d.name.startswith("MB2022"):
-                    print(d.stem)
-            return False
 
-        return super().check_options(opt)
+        # Check given B2 errors ---
+        exists = False
+        if accel.b2_errors is not None: 
+            # check if user gave absolute path (! with_suffix does not work, due to '.' in name)
+            exists = Path(f"{accel.b2_errors!s}.errors").is_file()
+
+        if not exists:
```

Comment:
If `exists` is a `bool` I would prefer `is True/False` comparisons

Reply:
What?

Reply:
Here `exists` is a boolean. I would prefer this check to be `if exists is False`.